MetaMask/metamask-extension
# Pass the name of the blocking phishing provider  to the phishing warning page

## Explanation

With version 4.0.0 of the PhishingController, we now support multiple phishing providers and can inspect which of these providers flagged the current page as phishing. This pull request captures the name of the provider, and forwards it to the phishing warning page.

This will allow the warning page to display attribution to who issued the block, and also allow for us to point the `report an issue` link to the correct location.


## Manual Testing Steps

* Visit test.metamaskphishing.io
* You should now be blocked by the phishing page
* In your URL you should see the following as a URL parameter

```
phishingProviderName=MetaMask
```

## Pre-merge author checklist


## Pre-merge reviewer checklist
